katharsis-core/src/test/java/io/katharsis/resource/mock/models/Task.java

```diff
@@ -77,6 +73,15 @@ public void setName(@SuppressWarnings("SameParameterValue") String name) {
         this.name = name;
     }
 
+    public String getCategory() {
+        return category;
+    }
+
+    public void setCategory(String category) {
+    	if(category == null) throw new IllegalArgumentException("Category cannot be set to null!");
```

Comment:
As these are only example classes, this is fine. But really, a class just for testing should be created.